docs/developers/basics/creating-an-addon.md

```diff
@@ -45,6 +45,59 @@ See [this source](https://wiki.facepunch.com/gmod/Lua_Folder_Structure) for a de
 
 ## Publishing your Addon
 
+## Windows
+
+In order to upload your addon to the Steam Workshop, you need to first compile it into a .gma file.
+
+1. Your addon file must contain an "addon.json" within the `<addon name>/` directory of your folder. This consists of a key:value pair table with information used by the Steam 
+
+    Workshop to create tags for uploads.
+    It should have a structure like this:
```

Comment:
i think this is misformatted?

Reply:
On the new commits I think I've fixed what you're referring to. I'm trying to span the numbered list over the entire publishing section